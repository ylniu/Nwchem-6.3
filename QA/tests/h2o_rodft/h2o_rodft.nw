echo
start h2o_rodft

memory global 40 mb stack 23 mb heap 5 mb

charge 1

geometry units au
 O 0       0        0
 H 0       1.430   -1.107
 H 0      -1.430   -1.107
end

basis
  O library 6-31g*
  H library 6-31g*
end

dft
  rodft
  mult 2
  cgmin
end

task dft
