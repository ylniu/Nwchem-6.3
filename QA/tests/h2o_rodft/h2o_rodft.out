 argument  1 = h2o_rodft.nw
 


============================== echo of input deck ==============================
echo
start h2o_rodft

memory global 40 mb stack 23 mb heap 5 mb

charge 1

geometry units au
 O 0       0        0
 H 0       1.430   -1.107
 H 0      -1.430   -1.107
end

basis
  O library 6-31g*
  H library 6-31g*
end

dft
  rodft
  mult 2
  cgmin
end

task dft
================================================================================


                                         
                                         
 
 
              Northwest Computational Chemistry Package (NWChem) 6.0
              ------------------------------------------------------
 
 
                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352
 
                              Copyright (c) 1994-2010
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute
 
             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file
 
                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname      = orion
    program       = ../../../bin/LINUX64/nwchem
    date          = Wed Jan 12 10:37:43 2011

    compiled      = Wed_Jan_12_10:20:28_2011
    source        = /home/niri/nwchem/nwchem
    nwchem branch = Development
    input         = h2o_rodft.nw
    prefix        = h2o_rodft.
    data base     = ./h2o_rodft.db
    status        = startup
    nproc         =        1
    time left     =     -1s



           Memory information
           ------------------

    heap     =     655361 doubles =      5.0 Mbytes
    stack    =    3014657 doubles =     23.0 Mbytes
    global   =    5242880 doubles =     40.0 Mbytes (distinct from heap & stack)
    total    =    8912898 doubles =     68.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------
 
  0 permanent = .
  0 scratch   = .
 
 
 
 
                                NWChem Input Module
                                -------------------
 
 
 C2V symmetry detected

          ------
          auto-z
          ------
 
 
                             Geometry "geometry" -> ""
                             -------------------------
 
 Output coordinates in a.u. (scale by  1.000000000 to convert to a.u.)
 
  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 O                    8.0000     0.00000000     0.00000000     0.22140000
    2 H                    1.0000    -1.43000000     0.00000000    -0.88560000
    3 H                    1.0000     1.43000000     0.00000000    -0.88560000
 
      Atomic Mass 
      ----------- 
 
      O                 15.994910
      H                  1.007825
 

 Effective nuclear repulsion energy (a.u.)       9.1971984402

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000
 
      Symmetry information
      --------------------
 
 Group name             C2v       
 Group number             16
 Group order               4
 No. of unique centers     2
 
      Symmetry unique atoms
 
     1    2
 


                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles
 
      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       0.95697
    2 Stretch                  1     3                       0.95697
    3 Bend                     2     1     3               104.51124
 
 
            XYZ format geometry
            -------------------
     3
 geometry
 O                     0.00000000     0.00000000     0.11715984
 H                    -0.75672347     0.00000000    -0.46863937
 H                     0.75672347     0.00000000    -0.46863937
 
 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |       a.u.
 ------------------------------------------------------------------------------
    2 H                |   1 O                |     1.80841  |     1.80841
    3 H                |   1 O                |     1.80841  |     1.80841
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          2
 ==============================================================================



 ==============================================================================
                                 internuclear angles
 ------------------------------------------------------------------------------
        center 1       |       center 2       |       center 3       |  degrees
 ------------------------------------------------------------------------------
    2 H                |   1 O                |   3 H                |   104.51
 ------------------------------------------------------------------------------
                            number of included internuclear angles:          1
 ==============================================================================



  library name resolved from: environment
  library file name is: </home/niri/nwchem/nwchem/src/basis/libraries/>
  
                      Basis "ao basis" -> "" (cartesian)
                      -----
  O (Oxygen)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  5.48467170E+03  0.001831
  1 S  8.25234950E+02  0.013950
  1 S  1.88046960E+02  0.068445
  1 S  5.29645000E+01  0.232714
  1 S  1.68975700E+01  0.470193
  1 S  5.79963530E+00  0.358521
 
  2 S  1.55396160E+01 -0.110778
  2 S  3.59993360E+00 -0.148026
  2 S  1.01376180E+00  1.130767
 
  3 P  1.55396160E+01  0.070874
  3 P  3.59993360E+00  0.339753
  3 P  1.01376180E+00  0.727159
 
  4 S  2.70005800E-01  1.000000
 
  5 P  2.70005800E-01  1.000000
 
  6 D  8.00000000E-01  1.000000
 
  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.87311370E+01  0.033495
  1 S  2.82539370E+00  0.234727
  1 S  6.40121700E-01  0.813757
 
  2 S  1.61277800E-01  1.000000
 


 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 O                           6-31g*                  6       15   3s2p1d
 H                           6-31g*                  2        2   2s


 
                                 NWChem DFT Module
                                 -----------------
 
 
  Caching 1-el integrals 
 
            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  spin polarized.
          No. of atoms     :     3
          No. of electrons :     9
           Alpha electrons :     5
            Beta electrons :     4
          Charge           :     1
          Spin multiplicity:     2
          Use of symmetry is: on ; symmetry adaption is: on 
          Maximum number of iterations:  30
          AO basis - number of functions:    19
                     number of shells:    10
          Convergence on energy requested: 1.00D-06
          Convergence on density requested: 1.00D-05
          Convergence on gradient requested: 5.00D-04
 
              XC Information
              --------------
                        Slater Exchange Functional  1.000 local    
                      VWN V Correlation Functional  1.000 local    
 
             Grid Information
             ----------------
          Grid used for XC integration:  medium    
          Radial quadrature: Mura-Knowles        
          Angular quadrature: Lebedev. 
          Tag              B.-S. Rad. Rad. Pts. Rad. Cut. Ang. Pts.
          ---              ---------- --------- --------- ---------
          O                   0.60       49           5.0       434
          H                   0.35       45           6.0       434
          Grid pruning is: on 
          Number of quadrature shells:    94
          Spatial weights used:  Erf1
 
          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL): 1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         30 iters            30 iters 

 
      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho: 1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul: 1.00D-08

 
      Superposition of Atomic Density Guess
      -------------------------------------
 
 Sum of atomic energies:         -75.75081731

 Renormalizing density from      10.00 to      9
 
      Non-variational initial energy
      ------------------------------

 Total energy =     -70.704093
 1-e energy   =    -109.563990
 2-e energy   =      29.662699
 HOMO         =      -1.167537
 LUMO         =      -0.290539
 
 
      Symmetry analysis of molecular orbitals - initial
      -------------------------------------------------
 
  Numbering of irreducible representations: 
 
     1 a1          2 a2          3 b1          4 b2      
 
  Orbital symmetries:
 
     1 a1          2 a1          3 b1          4 a1          5 b2      
     6 a1          7 b1          8 a1          9 b2         10 b1      
    11 a1         12 b1         13 a1         14 a1         15 a2      
 


 ----------------------------------------------
         Quadratically convergent ROHF

 Convergence threshold     :          5.000E-04
 Maximum no. of iterations :           30
 Final Fock-matrix accuracy:          1.000E-07
 ----------------------------------------------


 #quartets = 1.009D+03 #integrals = 5.756D+03 #direct =  0.0% #cached =100.0%


 Integral file          = ./h2o_rodft.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   = 203886
 No. of bits per label  =      8        No. of bits per value  =     64


 Grid_pts file          = ./h2o_rodft.gridpts.0
 Record size in doubles =  12289        No. of grid_pts per rec  =   3070
 Max. records in memory =     16        Max. recs in file   =   1087309


              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1      -75.2285252583  2.00D+00  8.76D-01      0.1
                 2      -75.3729083055  2.80D-01  1.45D-01      0.1
                 3      -75.3764036069  1.14D-01  6.99D-02      0.1
                 4      -75.3766983207  3.28D-02  1.55D-02      0.2
                 5      -75.3767418857  1.52D-02  7.46D-03      0.3
                 6      -75.3767545662  2.62D-03  1.38D-03      0.4
                 7      -75.3767549125  1.17D-03  4.64D-04      0.5
                 8      -75.3767550156  2.89D-04  1.55D-04      0.6


         Total DFT energy =      -75.376755015644
      One electron energy =     -117.982720306307
           Coulomb energy =       41.781439104184
    Exchange-Corr. energy =       -8.372672253719
 Nuclear repulsion energy =        9.197198440198

 Numeric. integr. density =        9.000001185074

     Total iterative time =      0.6s


 
                       DFT Final Molecular Orbital Analysis
                       ------------------------------------
 
 Vector    2  Occ=2.000000D+00  E=-1.411545D+00  Symmetry=a1
              MO Center=  4.6D-17, -6.3D-17, -8.9D-02, r^2= 4.6D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      0.499998  1 O  s                  6      0.427447  1 O  s          
     1     -0.214706  1 O  s                  5     -0.151899  1 O  pz         
 
 Vector    3  Occ=2.000000D+00  E=-9.432815D-01  Symmetry=b1
              MO Center=  3.1D-16,  2.9D-17, -6.3D-02, r^2= 6.5D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.588107  1 O  px                 7      0.286583  1 O  px         
    16     -0.221919  2 H  s                 18      0.221919  3 H  s          
 
 Vector    4  Occ=2.000000D+00  E=-7.941256D-01  Symmetry=a1
              MO Center=  2.6D-17,  2.2D-16,  2.3D-01, r^2= 5.7D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.619682  1 O  pz                 9      0.354399  1 O  pz         
     6      0.277344  1 O  s                  2      0.200233  1 O  s          
 
 Vector    5  Occ=1.000000D+00  E=-7.566444D-01  Symmetry=b2
              MO Center=  3.2D-17, -2.3D-16,  9.5D-02, r^2= 5.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.725083  1 O  py                 8      0.414093  1 O  py         
 
 Vector    6  Occ=0.000000D+00  E=-2.924452D-01  Symmetry=a1
              MO Center=  4.1D-15, -9.6D-17, -6.0D-01, r^2= 2.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.979955  1 O  s                 19     -0.848003  3 H  s          
    17     -0.848003  2 H  s                  9     -0.344911  1 O  pz         
     5     -0.317023  1 O  pz                 2      0.250540  1 O  s          
    18     -0.183203  3 H  s                 16     -0.183203  2 H  s          
 
 Vector    7  Occ=0.000000D+00  E=-2.171821D-01  Symmetry=b1
              MO Center= -4.2D-15,  3.0D-31, -5.2D-01, r^2= 2.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    17      1.101473  2 H  s                 19     -1.101473  3 H  s          
     7      0.603267  1 O  px                 3      0.485465  1 O  px         
    16      0.236493  2 H  s                 18     -0.236493  3 H  s          
 
 Vector    8  Occ=0.000000D+00  E= 3.534060D-01  Symmetry=b1
              MO Center=  3.1D-16, -2.7D-20, -1.5D-01, r^2= 1.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    16      0.792701  2 H  s                 18     -0.792701  3 H  s          
    19      0.758144  3 H  s                 17     -0.758144  2 H  s          
    12      0.402442  1 O  dxz                7      0.389985  1 O  px         
 
 Vector    9  Occ=0.000000D+00  E= 3.817697D-01  Symmetry=b2
              MO Center= -5.1D-17,  6.7D-15,  1.1D-01, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      1.078059  1 O  py                 4     -0.900164  1 O  py         
 
 Vector   10  Occ=0.000000D+00  E= 3.869959D-01  Symmetry=a1
              MO Center= -6.2D-17, -6.7D-15,  1.9D-01, r^2= 1.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.811076  1 O  pz                 9     -0.730415  1 O  pz         
    16      0.474834  2 H  s                 18      0.474834  3 H  s          
    19     -0.472364  3 H  s                 17     -0.472364  2 H  s          
     6     -0.223367  1 O  s                 10      0.193582  1 O  dxx        
     2      0.169209  1 O  s          
 
 Vector   11  Occ=0.000000D+00  E= 4.538923D-01  Symmetry=a1
              MO Center=  7.6D-17,  1.7D-16, -1.1D-01, r^2= 1.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.946046  1 O  s                  9      0.742458  1 O  pz         
    16      0.699012  2 H  s                 18      0.699012  3 H  s          
     2     -0.630148  1 O  s                 19     -0.495048  3 H  s          
    17     -0.495048  2 H  s                 13     -0.315267  1 O  dyy        
     5     -0.315185  1 O  pz                15     -0.189617  1 O  dzz        
 
 Vector   12  Occ=0.000000D+00  E= 5.919439D-01  Symmetry=b1
              MO Center=  3.1D-17, -1.2D-32,  2.4D-02, r^2= 1.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      1.713054  1 O  px                19     -1.102336  3 H  s          
    17      1.102336  2 H  s                  3     -0.927128  1 O  px         
 
 Vector   13  Occ=0.000000D+00  E= 7.313427D-01  Symmetry=a1
              MO Center=  5.6D-17,  3.7D-17, -3.7D-01, r^2= 1.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      3.706307  1 O  s                  2     -1.533406  1 O  s          
     9     -1.001447  1 O  pz                19     -0.895921  3 H  s          
    17     -0.895921  2 H  s                 10     -0.648578  1 O  dxx        
    15     -0.359621  1 O  dzz                5      0.329795  1 O  pz         
    18     -0.245137  3 H  s                 16     -0.245137  2 H  s          
 
 Vector   14  Occ=0.000000D+00  E= 1.175111D+00  Symmetry=a2
              MO Center=  3.6D-16, -3.0D-17,  1.2D-01, r^2= 6.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    11      1.732051  1 O  dxy        
 
 Vector   15  Occ=0.000000D+00  E= 1.183936D+00  Symmetry=a1
              MO Center= -1.3D-17, -1.7D-17,  1.8D-01, r^2= 6.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    15      1.005619  1 O  dzz               10     -0.562866  1 O  dxx        
    13     -0.393664  1 O  dyy                6     -0.243762  1 O  s          
     9      0.153472  1 O  pz         
 

 Parallel integral file used       1 records with       0 large values


 Task  times  cpu:        0.6s     wall:        1.2s
 
 
                                NWChem Input Module
                                -------------------
 
 
 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls: 1008     1008     2.83e+04 2686     1.61e+04  195        0        0     
number of processes/call 1.00e+00 1.00e+00 1.00e+00 1.00e+00 0.00e+00
bytes total:             1.15e+07 1.78e+06 5.39e+06 4.63e+05 0.00e+00 0.00e+00
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 112400 bytes
 
MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        22	        50
	current total bytes		         0	         0
	maximum total bytes		   2633856	  22509584
	maximum total K-bytes		      2634	     22510
	maximum total M-bytes		         3	        23
 
 
                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:
 
                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018
 
                              AUTHORS & CONTRIBUTORS
                              ----------------------
      E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski, T. P. Straatsma,
     M. Valiev, H. J. J. van Dam, D. Wang, E. Apra, T. L. Windus, J. Hammond,
     J. Autschbach, P. Nichols, S. Hirata, M. T. Hackler, Y. Zhao, P.-D. Fan,
      R. J. Harrison, M. Dupuis, D. M. A. Smith, K. Glaesemann, J. Nieplocha,
      V. Tipparaju, M. Krishnan, A. Vazquez-Mayagoitia, L. Jensen, M. Swart,
      Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown,
             G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza, K. Hirao,
        R. Kendall, J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell,
       D. Bernholdt, P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. Deegan,
        K. Dyall, D. Elwood, E. Glendening, M. Gutowski, A. Hess, J. Jaffe,
        B. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield,
    X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing, G. Sandrone,
       M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. Wong, Z. Zhang.

 Total times  cpu:        0.6s     wall:        1.4s
